src/dependencies.ts

```diff
@@ -0,0 +1,65 @@
+import { Octokit } from 'octokit';
```

Comment:
Why are you using the `import * as X` syntax? Why not just `import X from 'X'` ?

Reply:
In my opinion, for the database operations, it helps me understand which methods require a local or remote handler.
As for the external libraries, I think we can drop the renames.